chapters/fr/_toctree.yml

```diff
@@ -42,3 +42,75 @@
   - local: chapter5/8
     title: Quiz de fin de chapitre
     quiz: 5
+
+- title: 6. La bibliothèque 🤗 Tokenizer
+  sections:
+  - local: chapitre6/1
```

Comment:
Ah sorry the instructions weren't super clear: we need to keep the filenames in English due to the way our frontend works. So all these `local` fields should be in English with e.g. `chapter6/1` here and so on

Reply:
It is modified :)

Reply:
Erf it did not allow to pass the test :'(

Reply:
Thanks for the fix!